pkg/controller/certificaterequests/sync_test.go

```diff
@@ -159,12 +161,23 @@ func TestSync(t *testing.T) {
 			Type:               cmapi.CertificateRequestConditionReady,
 			Status:             cmapi.ConditionFalse,
 			Reason:             "Pending",
-			Message:            "Referenced Issuer not found",
+			Message:            `referenced "Issuer" not found: issuer.certmanager.k8s.io "fake-issuer" not found`,
+			LastTransitionTime: &nowMetaTime,
+		}),
+	)
+
+	exampleCRIssuerUnknownTypeFoundPendingCondition := gen.CertificateRequestFrom(exampleCR,
+		gen.SetCertificateRequestStatusCondition(cmapi.CertificateRequestCondition{
+			Type:               cmapi.CertificateRequestConditionReady,
+			Status:             cmapi.ConditionFalse,
+			Reason:             "Pending",
+			Message:            "failed to obtain referenced issuer type: no issuer specified for Issuer 'default-unit-test-ns/fake-issuer'",
```

Comment:
This error message is a bit confusing.. maybe "no issuer type specified" (i.e. add `type`?)